src/pages/blocks/Blocks.tsx

```diff
@@ -118,7 +118,7 @@ const Blocks: React.FC<MatchParams> = props => {
   const columns =
     width > 768
       ? [
-          { name: 'Platform', accessor: 'platform' },
+          { name: 'Network', accessor: 'platform' },
```

Comment:
I did not change the `accessor` on purpose because of some feedback on discord
> Changing it makes sense as long as there is a consideration that we will need to change it back if we ever add another protocol